dashboard/app/models/sections/clever_section.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: sections
+#
+#  id                :integer          not null, primary key
+#  user_id           :integer          not null
+#  name              :string(255)
+#  created_at        :datetime
+#  updated_at        :datetime
+#  code              :string(255)
+#  script_id         :integer
+#  course_id         :integer
+#  grade             :string(255)
+#  login_type        :string(255)      default("email"), not null
+#  deleted_at        :datetime
+#  stage_extras      :boolean          default(FALSE), not null
+#  section_type      :string(255)
+#  first_activity_at :datetime
+#  pairing_allowed   :boolean          default(TRUE), not null
+#
+# Indexes
+#
+#  fk_rails_20b1e5de46        (course_id)
+#  index_sections_on_code     (code) UNIQUE
+#  index_sections_on_user_id  (user_id)
+#
+
+class CleverSection < OmniAuthSection
+  def self.from_service(course_id, owner_id, student_list)
+    code = "C-#{course_id}"
+
+    students = student_list.map do |student|
+      data = student['data']
+      OmniAuth::AuthHash.new(
+        uid: data['id'],
+        provider: 'clever',
+        info: {
+          name: data['name'],
+          dob: data['dob'],
+        },
+      )
+    end
+
+    OmniAuthSection.from_omniauth(
```

Comment:
nit: you don't need the classname here, since it's in the inheritance hierarchy.